app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Service catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external services. This catalog is the single source of truth for your organization’s service inventory and their dependencies.
+
+{% mermaid %}
+graph TD
+    subgraph "Built-In Integrations"
+        D[{{site.base_gateway}}]
+        E[{{site.mesh_product_name}}]
+        F[{{site.konnect_short_name}} Analytics]
+        G[API Products]
+    end
+
+    subgraph "External Integrations"
+        H[GitHub]
+        I[PagerDuty]
+    end
+
+    D -->A[{{site.konnect_short_name}} Service Catalog]
+    E -->A
+    F -->A
+    G -->A
+
+    H -->A
+    I -->A
+{% endmermaid %}
+
+> Figure 1: This diagram shows how you can pull services from built-in integrations (like {{site.base_gateway}}, {{site.mesh_product_name}}, {{site.konnect_short_name}} Analytics, and API Products) and external services (like from GitHub and PagerDuty) into the Service Catalog. This consolidates all your organization's services and service health metrics into a single view.
+
+Specifically, Service Catalog addresses the following problems:
+
+* **Service discovery:** Gain visibility into all your services, including unrecognized or undiscovered APIs that live in your organization’s fragmented ecosystems. 
+* **Service ownership:** Keep track of your organization's service ownership by mapping teams and services. 
+* **Service health:** The Service Catalog consolidates key service health metrics into a single view, so you no longer need to check multiple tools (like GitHub and PagerDuty) for recent activity and status. It also helps you quickly identify and reuse existing services, reducing duplication and effort.
+* **Service governance and policy enforcement:** Enables governance in how services are created and maintained across your company to ensure you adhere to security, compliance, and engineering best practices.
+
+{% mermaid %}
+graph TD
+    subgraph "{{site.konnect_short_name}} Service Catalog"
+        SC[Service Catalog]
+        SG[{{site.base_gateway}} Service]
+        API[API Product]
+    end
+
+    subgraph "External APIs"
+        GitHub[GitHub]
+        PagerDuty[PagerDuty]
+        Data[External API Data]
+    end
+
+    GitHub -->|Pulls Service| SC
+    PagerDuty -->|Pulls Service| SC
+    SC -->|Maps to| SG
```

Comment:
Let's talk about this more tomorrow, but a Service Catalog Service doesn't quite technically "map to" a Gateway Service. It's more of the other way around, actually. A Gateway Service is the Resource type that we ingest from the Gateway Manager Integration. A Service Catalog Service can be thought of as nothing but an aggregation of Resources (Gateway Service, Mesh Service, GitHub Repository, Datadog Monitor, Slack Channel, etc.) along with additional metadata attached (team owner, product manager, etc.).